src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/Interfaces/ITestSession.cs

```diff
@@ -195,6 +195,17 @@ public interface ITestSession : IDisposable, ITestSessionAsync
     [Obsolete("This API is not final yet and is subject to changes.", false)]
     bool StopTestSession(ITestSessionEventsHandler eventsHandler);
 
+    /// <summary>
+    /// Stops the test session.
+    /// </summary>
+    ///
+    /// <param name="options">Test Platform options.</param>
+    /// <param name="eventsHandler">The session event handler.</param>
+    /// 
+    /// <returns>True if the session was successfuly stopped, false otherwise.</returns>
+    [Obsolete("This API is not final yet and is subject to changes.", false)]
```

Comment:
mmm weird Obsolete for a new api? Preview api(in preview package) are subject to change by design, I think it's not needed.

Reply:
As discussed with Artur, we want to be extra clear those APIs shouldn't be used yet. They already went through a few release cycles so they are out there, but this is a disclaimer we won't necessarily stand by them in their current form. When the API takes its final form we will remove the `Obsolete` attribute.